xml/deployment_ganesha.xml

```diff
@@ -526,26 +525,26 @@ CACHEINODE {
     NParts = 1;
     Cache_Size = 1;
 }
-EXPORT
-{
-    Export_ID=100;
-    Protocols = 4;
-    Transports = TCP;
-    Path = /;
-    Pseudo = /ceph/;
-    Access_Type = RW;
-    Attr_Expiration_Time = 0;
-    FSAL {
-        Name = CEPH;
-    }
-}
 RADOS_KV
 {
-    pool = "cephfs_metadata";
-    namespace = "ganesha";
-    #nodeid = "a";
+    pool = "<replaceable>rados_pool</replaceable>";
+    namespace = "<replaceable>pool_namespace</replaceable>";
+    nodeid = "<replaceable>fqdn</replaceable>"
+    UserId = "<replaceable>cephx_user_id</replaceable>";
+    Ceph_Conf = "<replaceable>path_to_ceph.conf</replaceable>"
 }
 </screen>
+   <para>
+   You can find out the values for <replaceable>rados_pool</replaceable> and
+   <replaceable>pool_namespace</replaceable> by checking the already existing line in
+   the configuration of the form
+   <screen>%url rados://<replaceable>rados_pool</replaceable>/<replaceable>pool_namespace</replaceable>/...
+   </screen>.
```

Comment:
Sorry, I didn't see this before. It's complaing about this. You can't have a screen tag within a <para>.
So close the para after "form" and open again when you start with "The value"

Reply:
Thanks for the help. Pushed the fix